packages/flutter/lib/src/services/text_input.dart

```diff
@@ -721,10 +721,34 @@ TextAffinity? _toTextAffinity(String? affinity) {
   return null;
 }
 
-/// A floating cursor state the user has induced by force pressing an iOS
-/// keyboard.
+/// The state of a "floating cursor" drag on an iOS soft keyboard.
+///
+/// The "floating cursor" cursor-positioning mode is an iOS feature used to
+/// precisely position the caret in some editable text. When the user
```

Comment:
nit: force touch/long press/two-finger pan also picks up the cursor so maybe:
some editable text using certain touch gestures.  As an example: when the user

Reply:
Good call!